src/yunohost/app.py

```diff
@@ -993,16 +994,41 @@ def app_remove(operation_logger, app):
     os.system('chown -R admin: /tmp/yunohost_remove')
     os.system('chmod -R u+rX /tmp/yunohost_remove')
 
-    args_list = [app]
+    with open(os.path.join(APPS_SETTING_PATH, app, 'manifest.json')) as json_manifest:
+        manifest = json.load(json_manifest)
 
-    env_dict = {}
+    args_dict = {} if not args else \
+        dict(urlparse.parse_qsl(args, keep_blank_values=True))
+
+    # Retrieve arguments list for remove script
+    if "remove" in manifest.get("arguments", {}) and manifest["arguments"]["remove"]:
+        args_odict = _parse_args_from_manifest(manifest, 'remove', args=args_dict)
+        args_list = [value[0] for value in args_odict.values()]
+        args_list.append(app)
```

Comment:
Uh I have no idea what I'm talking about but shouldn't `app` be in first position so that it corresponds to `$1` ? (Well I guess anyway all recents apps likely to require argument for remove don't rely on `$1, $2, ...` anymore)